src/modelinstance.hpp

```diff
@@ -42,6 +42,11 @@
 #include "status.hpp"
 #include "tensorinfo.hpp"
 
+namespace inference {
```

Comment:
Why do we need this? Isnt it properly defined in KServe proto?

Reply:
This is forward declaration to avoid header inclusion.